src/blk/aio/aio.cc

```diff
@@ -26,7 +26,7 @@ int aio_queue_t::submit_batch(aio_iter begin, aio_iter end,
 
   aio_iter cur = begin;
 #if defined(HAVE_LIBAIO)
-  struct aio_t *piocb[max_iodepth];
+  std::vector<struct aio_t*> piocb(max_iodepth);
```

Comment:
This is the core I/O path -- consider a `small_vector` of 1024 (`bdev_aio_max_queue_depth` default) here.